ci/sge/docker-compose.yml

```diff
@@ -41,8 +41,6 @@ services:
     build:
       context: .
       target: slave
-      args:
-        PYTHON_VERSION: 3.9
```

Comment:
I'm not sure why this was specified for the `slave2` container but not `slave1` or `master`. So removing it here.